src/ResourceManager/HDInsight/Commands.HDInsight.Test/UnitTests/ConfigurationTests.cs

```diff
@@ -51,7 +51,7 @@ public void CanCreateNewConfig()
                     f.WriteObject(
                         It.Is<AzureHDInsightConfig>(
                             c =>
-                                c.ClusterType == HDInsightClusterType.Hadoop &&
+                                c.ClusterType == "Hadoop" &&
```

Comment:
Previously we have an Enum, but have changed the contracts to accept any string, which will be processed by the back end. All but one instance of "Hadoop" is in the test code and don't need to be "Hadoop." Is using constant in this case?